source/geometry/FloatConvexHull2D.ooc

```diff
@@ -24,6 +24,13 @@ FloatConvexHull2D: class {
 		if (computeHull)
 			this computeHull()
 	}
+	init: func ~fromArray (array: FloatPoint2D[], computeHull := true) {
+		this _points = VectorList<FloatPoint2D> new(array length)
+		for (i in 0 .. array length)
+			this _points add(array[i])
+		if (computeHull)
+			this computeHull()
```

Comment:
That sounds reasonable. But I really don't feel like wrestling with ooc right now, so let's just add it here where we need it right now